main/ertp/api/amount-math.md

```diff
@@ -1,189 +1,304 @@
+kind your contract uses. The second parameter, `MathKind` is optional and 
+defaults to `MathKind.NAT` if not given. For example
+```js
+import { MathKind, makeIssuerKit } from '@agoric/ertp';
+makeIssuerKit('quatloos`); // Defaults to 'MathKind.NAT'
+makeIssuerKit('foobars', 'MathKind.STRING_SET');
+makeIssuerKit('kitties', MathKind.SET');
+```
 
 ## Amount
 
-Amounts are descriptions of digital assets, answering the questions "how much" and "of what kind". Amounts are values labeled with a brand. AmountMath executes the logic of how amounts are changed when digital assets are merged, separated, or otherwise manipulated. For example, a deposit of 2 bucks into a purse that already has 3 bucks gives a new balance of 5 bucks. An empty purse has 0 bucks. AmountMath relies heavily on polymorphic MathHelpers, which manipulate the unbranded portion.
+An `amount` is a description of digital assets, answering the questions "how much?" and "of what kind?". It is a `value` ("how much") labeled with a `brand` ("of what kind"). `AmountMath` executes the logic of how an `amount` changes when digital assets are merged, separated, or otherwise manipulated. For example, a deposit of 2 Quatloos into a `purse` that already has 3 Quatloos gives a new balance of 5 Quatloos. An empty `purse` has 0 Quatloos. 
 
 ```js
 someAmount: {
-  brand,
-  value: someValue
+  brand: someBrand,
+  value: someValue,
 }
 ```
 
 ## Value
 
-Values describe how much of something that can be owned or shared. Fungible values are normally represented by natural numbers. Other values may be represented as strings naming a particular right, or an arbitrary object that sensibly represents the rights at issue.
+`values` describe how much of something can be owned or shared. A fungible `value` is normally represented by a natural number. Other `values` may be represented as strings naming a particular right, or an arbitrary object that sensibly represents the rights at issue.
+
+A `value` must be `Comparable`.
+
+## LocalAmountMath
+
+We encourage you to make and use `LocalAmountMath`, a local and thus synchronous version of `AmountMath`. 
+Their local or remote status is the only different between the two types; each has the same methods, the
+same kinds (`MathKind.NAT`, etc.) and the same one-to-one relationship with a `mint`, `issuer`, and `brand`.
+If necessary, both a `LocalAmountMath` and an `AmountMath` can exist and be associated with the same
+`mint`, `issuer`, and `brand` **tyg todo This last is a guess on my part. Correct?**.
 
-Value must be Comparable.
+## makeLocalAmountMath(issuer)
+- `issuer`: `{issuer}`
+Returns: `{ LocalMathAmount }`
+
+Creates and returns a local (synchronous) Amount Math object. The new `localAmountMath` has a one-to-one
+relationship with the `issuer`, and thus also to the `brand` and `mint` in one-to-one
+```js
+import { makeLocalAmountMath } from '@agoric/ertp';
+const quatloosLocalAmountMath = await makeLocalAmountMath(quatloosIssuer);
```

Comment:
users wouldn't name it with "local". We've designed the API so the only kind of AmountMath you can get is a local one. Perhaps we should have left the "local" out of the function's name, but we wanted to call attention to the fact that it has to do some asynchronous operations to get the Brand, and that's why you have to include the "await".